modules/osd-persistent-storage-csi-efs-sts.adoc

```diff
@@ -124,27 +124,25 @@ Save the output. You will use it in the next steps.
 +
 [source,terminal]
 ----
-$ aws iam create-policy \
+POLICY=$(aws iam create-policy \
   --policy-name "<your_rosa_cluster_name>-rosa-efs-csi" \
   --policy-document file://<your_policy_file_name>.json \
   --query 'Policy.Arn' --output text) || \
   POLICY=$(aws iam list-policies \
   --query 'Policies[?PolicyName==`rosa-efs-csi`].Arn' \
-  --output text
+  --output text)
```

Comment:
Maybe I missed something, but why the follow cmd is not enough to get the policy_ARN?

```
POLICY=$(aws iam create-policy \
  --policy-name "<your_rosa_cluster_name>-rosa-efs-csi" \
  --policy-document file://<your_policy_file_name>.json \
  --query 'Policy.Arn' --output text)
```

Reply:
If possible can we modify line no:115 like this? 
ROLE=$(aws iam create-role \
  --role-name "<your_cluster_name>-aws-efs-csi-operator" \
  --assume-role-policy-document file://<your_trust_file_name>.json \
  --query "Role.Arn" --output text)
echo $ROLE

Reply:
@ropatil010 Modified Line 115 as per your suggestion.